validphys2/src/validphys/core.py

```diff
@@ -243,19 +243,12 @@ def grid_values_index(self):
         -----
         The range object can be used efficiently as a Pandas index.
         """
-        if self.error_type == "replicas":
-            return range(1, len(self))
-        elif self.error_type in ("hessian", "symmhessian"):
-            return range(0, len(self))
-        else:
-            raise RuntimeError(f"Unknown error type: {self.stats_class}")
+        return range(0, len(self))
```

Comment:
aii, this is a bit confusing with making rep1 the 0 index, but I guess it's libNNPDF stuff.. Anyway, I guess the docstring needs to be adjusted.

Reply:
AFAICT rep0 would be the index 0, not rep1.
Also we should check if we still need this method now that it is trivial.